### ISSUE TITLE
Missing metadata in published package prevents usage with Angular 9

### ISSUE THREAD

**Issue by luchsamapparat** [visjs/ngx-vis#383]
## Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed
- [X] I'm reporting the issue to the correct repository

## Current Behavior

The published package is missing Angular metadata (`*.metadata.json`).

see https://unpkg.com/browse/ngx-vis@3.0.1/

## Expected Behavior

When building the project locally, `npm run build` already produces the correct output in `dist`:

```
bundles
components
esm2015
esm5
fesm2015
fesm5
CHANGELOG.md
LICENSE
ngxvis.d.ts
ngx-vis.d.ts
ngxvis.metadata.json
ngx-vis-3.0.1.tgz
package.json
README.md
```

## Failure Information

The missing Angular metadata prevents the library from being used with Angular 9 (tested with the latest RC):

```
ERROR in ../../node_modules/ngx-vis/ngx-vis.d.ts:8:22 - error TS-996002: Appears in the NgModule.imports of PlannerModule, but could not be resolved to an NgModule class

8 export declare class VisModule {
```

* Angular CLI: 9.0.0-rc.7
* Node: 12.14.0
* OS: win32 x64

**Comment by hypery2k** [comment]
thanks for reporting. seems an issue that within github actions to build is done before publishing

**Comment by hypery2k** [comment]
you can give `ngx-vis@3.0.1-build1578411729`: https://unpkg.com/browse/ngx-vis@3.0.1-build1578411729/

**Comment by hypery2k** [comment]
just published `3.0.2.`

**Comment by luchsamapparat** [comment]
Works 👍🏻
